resource-aggregate/cqrs/eventstore/eventstore.go

```diff
@@ -13,16 +13,28 @@ type VersionQuery struct {
 
 // SnapshotQuery used to load events from snapshot.
 type SnapshotQuery struct {
-	GroupID           string //filter by group ID
-	AggregateID       string //filter to certain aggregateID, groupID is required
-	SnapshotEventType string //required
+	GroupID     string //filter by group ID
+	AggregateID string //filter to certain aggregateID, groupID is required
 }
 
+type SaveStatus int
+
+const (
+	OK                   SaveStatus = 0  // events were stored
+	ConcurrencyException SaveStatus = 1  // need to reload events from store, because the versions of events are obsolete by concurrent store.
```

Comment:
Don't write in a description the solution what needs to be done. Describe what happened, the status.
`event with this version already exists`